### DISCUSSION TITLE
Error when I test my azure function with Prisma

### DISCUSSION THREAD

**Discussion by martinGM05** [prisma/prisma#24439] [Q&A]
### Question

When I deploy the azure function the function app show me some like this 
![image](https://github.com/prisma/prisma/assets/63875704/dd38e544-ff13-4145-a16f-bd1b958f0a20)
The message when I deploy is :
![image](https://github.com/prisma/prisma/assets/63875704/26bd61cd-0e71-4cdd-8b05-863ecfc4b7e2)

Prisma Schema: 
![image](https://github.com/prisma/prisma/assets/63875704/c5195efb-bc89-4350-9ca0-2667e58e79d9)

### Information about Prisma Schema, Client Queries and Environment (optional)

- OS: Windows
- Database: MySQL
- Node.js version: 20.11.0

-  Prisma: "@prisma/client": "^5.15.0",

**Comment by ludralph** [comment]
Hi @martinGM05 👋 

Thank you for raising this question.

Can you please provide the detailed error message you got? Please confirm that the Prisma Client is generated correctly during the build process. Sometimes, running `prisma generate` as part of the deployment script can help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
